### ISSUE TITLE
Error with `coiled run` command after updating to version `1.12.2`

### ISSUE THREAD

**Issue by andersy005** [coiled/feedback#272]
**Environment:**

```bash
❯ coiled --version                                                                                                                                                                                              
1.12.2
```

After updating Coiled to version 1.12.2, the following command, which worked in version 1.11.0, now fails:

```
❯ coiled run --container alpine --env FOO=BAR bash                                                                                                                                                              ```
```

```bash
Traceback (most recent call last):
  File "/Users/andersy005/mambaforge/envs/offsets-db-data/bin/coiled", line 10, in <module>
    sys.exit(cli())
  File "/Users/andersy005/mambaforge/envs/offsets-db-data/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/Users/andersy005/mambaforge/envs/offsets-db-data/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/Users/andersy005/mambaforge/envs/offsets-db-data/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/andersy005/mambaforge/envs/offsets-db-data/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/andersy005/mambaforge/envs/offsets-db-data/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/Users/andersy005/mambaforge/envs/offsets-db-data/lib/python3.10/site-packages/coiled/cli/run.py", line 402, in run
    start_run(
  File "/Users/andersy005/mambaforge/envs/offsets-db-data/lib/python3.10/site-packages/coiled/cli/run.py", line 449, in start_run
    runtime_env_dict = dict_from_key_val_list(env)
  File "/Users/andersy005/mambaforge/envs/offsets-db-data/lib/python3.10/site-packages/coiled/cli/run.py", line 248, in dict_from_key_val_list
    k, v = kv_list
ValueError: not enough values to unpack (expected 2, got 1)
```

It seems there's an issue with unpacking environment variables. This error did not occur in the previous version 1.11.0. Is this a known issue in version 1.12.2, or am I missing an update in the command syntax for the new version?

**Comment by ntabris** [comment]
Oh, sorry about that! That's a bug that got introduced recently, we'll push out a version with fix later today or tomorrow.

**Comment by ntabris** [comment]
@andersy005 thanks again for catching this! It's been fixed in [1.12.3](https://docs.coiled.io/user_guide/changelog.html#v1-12-3) (and there's now better test coverage).
